test/allCommandsNoJSON.nut.ts

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { TestSession, execCmd } from '@salesforce/cli-plugins-testkit';
+
+const projectPath = 'testProject_AllUserCommandsNoJSON';
+let session: TestSession;
+
+describe('verifies all commands run successfully (no json)', () => {
+  before(() => {
+    session = TestSession.create({
+      project: {
+        destinationDir: projectPath,
```

Comment:
You don't need to specify a destinationDir for the project here unless you really want to control that.  By default it will put it into the test session dir.

Reply:
gonna give that a try